src/Core.UnitTests/Notifications/NotificationServiceTests.cs

```diff
@@ -44,6 +45,12 @@ public void MefCtor_CheckIsExported()
                 MefTestHelpers.CreateExport<ILogger>());
         }
 
+        [TestMethod]
+        public void CheckIsNonSharedMefComponent()
+        {
+            MefTestHelpers.IsMefComponentNonShared(typeof(NotificationService));
```

Comment:
The method is returning a bool and this test doesn't contain any assertions so this test will always pass if the type has a `PartCreationPolicy`.

It might be better to structure the helper method like `CheckTypeCanBeImported` e.g. `CheckIsSingletonMefComponent` / `CheckIsNonSharedMefComponent`.